### DISCUSSION TITLE
Official Container Image?

### DISCUSSION THREAD

**Discussion by mpern** [opentofu/opentofu#733] [General]
Apologies in advance if this has been asked before and I just wasn't able to find it, but is there an official container/Docker image for OpenTofu?

Preferably a "verified" (or even "official"?) image at hub.docker.com, so I can just use it I'm my corporate pipelines 😄 

Ref:
https://hub.docker.com/r/hashicorp/terraform

**Comment by LanikSJ** [comment]
Looks like it's hosted on GitHub Container Registry: https://github.com/opentofu/opentofu/pkgs/container/opentofu and https://github.com/opentofu/opentofu/pkgs/container/opentofu/versions?filters%5Bversion_type%5D=tagged

**Reply by mpern** [reply]
Ah right 🤦‍♂️
